airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/manifest.yaml

```diff
@@ -2111,5 +2111,5 @@ streams:
 
 concurrency_level:
   type: ConcurrencyLevel
-  default_concurrency: 10
-  max_concurrency: 10
+  default_concurrency: 1
```

Comment:
The concern I have with having the concurrency to 1 is that if a partition generator can generate enough partitions to get the thread pool full, [it'll just sleep indefinitely](https://github.com/airbytehq/airbyte-python-cdk/blob/65e6a0d12e19a84db360053d3ec793c66b203512/airbyte_cdk/sources/streams/concurrent/partition_enqueuer.py#L58-L59) and we will have a deadlock. Do we have such streams that can generate a lot of partitions? Else, can we simply put this to 2 instead of 1 and that way, there should always be a partition reader reading from the thread pool and therefore removing stuck from the threadpool

Reply:
👍 , bumped to 2